src/pages/en/guides/publish-to-npm.md

```diff
@@ -111,21 +114,21 @@ The entry points allowed by Astro to import your component or any of its [files]
 }
 ```
 
-In this example, importing `my-component` would use `index.js`, while importing `my-component/astro` or `my-component/react` would use `MyAstroComponent.astro` or `MyReactComponent.jsx`.
+In this example, importing `my-component` would use `index.js`, while importing `my-component/astro` or `my-component/react` would use `MyAstroComponent.astro` or `MyReactComponent.jsx` respectively.
 
-#### `package.json#files`
+#### `files`
 
 ```json
 {
   "files": ["index.js", "MyAstroComponent.astro", "MyReactComponent.jsx"]
 }
 ```
 
```

Comment:
What do folks think about removing this? `files` is an optional optimisation to exclude unnecessary files from the bundle shipped via npm, but it’s also a footgun if a beginner gets all this set up, but then later adds a new file without updating this.